### ISSUE TITLE
Optional Main Database Port Options

### ISSUE THREAD

**Issue by kjaymiller** [kjaymiller/cookiecutter-relecloud#131]
We have support for changing the webport but not the main database port.

**Comment by pamelafox** [comment]
Is that needed? For PG at least, I've never had to change from 5432

**Comment by kjaymiller** [comment]
I agree that it doesn't really change but neither does the web port. I would say either have both of them configurable or neither of them configurable.

**Comment by kjaymiller** [comment]
I think it's okay to close this and perhaps make a new issue to remove the url route option (default `8000`) for consistency.
